repositories.bzl

```diff
@@ -12,24 +12,24 @@ VERSIONS = {
         "type": "github",
         "org": "bazelbuild",
         "repo": "rules_proto",
-        "ref": "84ba6ec814eebbf5312b2cc029256097ae0042c3",
-        "sha256": "3bce0e2fcf502619119c7cac03613fb52ce3034b2159dd3ae9d35f7339558aa3",
+        "ref": "f7a30f6f80006b591fa7c437fe5a951eb10bcbcf",
+        "sha256": "9fc210a34f0f9e7cc31598d109b5d069ef44911a82f507d5a88716db171615a8",
     },
     "com_google_protobuf": {
         # When updating, also update JS requirements, JS readme, Ruby requirements and C# requirements
```

Comment:
- I'm not sure what `JS readme` means here.
- `C# requirements` I'm not sure how to do this. The script I found is written to run on linux and I'm on mac.